templates/helper/class-based.js

```diff
@@ -1,6 +1,6 @@
 import Helper from "@ember/component/helper";
 
-export default class {{name.pascal}} extends Helper {
+export default class {{name.camel}} extends Helper {
```

Comment:
Bit weird to use camelCase for a class, but this seems to make more sense to me when using gts/gjs. Sames goes for modifiers.